### PR TITLE
docs: update k8s version to 1.19 >= and <= 1.21

### PR DESCRIPTION
**Pull request by varlogtim**

### CODE REVIEW

**Comment by vishnu2kmohan** [comment]
@varlogtim Just checking; do we need any updates to the helm charts to support K8s >=1.19 and <=1.21?

File: docs/how-to/installation/kubernetes.txt [hunk 1]
```diff
@@ -16,7 +16,8 @@ For general information about using Determined with Kubernetes, refer the
 Before installing Determined on a Kubernetes cluster, please ensure that the following prerequisites
 are satisfied:
 
--  The Kubernetes cluster should be running Kubernetes >= 1.15
+-  The Kubernetes cluster should be running Kubernetes versoin >= 1.19 and <= 1.21 (though later
```
Comment (vishnu2kmohan) [inline docs/how-to/installation/kubernetes.txt:19]:
```suggestion
-  The Kubernetes cluster should be running Kubernetes version >= 1.19 and <= 1.21, though later
```

**Review by vishnu2kmohan — CHANGES_REQUESTED** [review]
Minor nits, looks good otherwise!

File: docs/how-to/installation/kubernetes.txt [hunk 2]
```diff
@@ -16,7 +16,8 @@ For general information about using Determined with Kubernetes, refer the
...
+   versions may work)
```
Comment (vishnu2kmohan) [inline docs/how-to/installation/kubernetes.txt:20]:
```suggestion
   versions may work.
```

**Review by vishnu2kmohan — CHANGES_REQUESTED** [review]
One more minor suggestion.

**Comment by ioga** [comment]
By the way, our CI uses 1.18, so I'd expect it to work as well.

**Comment by varlogtim** [comment]
> @varlogtim Just checking; do we need any updates to the helm charts to support K8s >=1.19 and <=1.21?

Good catch @vishnu2kmohan . It actually looks like the helm chart links, at least in the generated docs.determined.ai, don't appear to work either.

File: docs/how-to/installation/setup-eks-cluster.txt [hunk 3]
```diff
@@ -238,7 +249,7 @@ file such as ``determined-autoscaler.yaml``:
              value: "true"
              effect: NoSchedule
          containers:
-           - image: k8s.gcr.io/autoscaling/cluster-autoscaler:v1.17.3
+           - image: <cluster-autoscaler-image>
```
Comment (eecsliu) [inline docs/how-to/installation/setup-eks-cluster.txt:252]: non-blocking: should we reiterate the link to autoscalers here? I have a feeling people might not read...

**Review by eecsliu — APPROVED** [review]
👩‍🍳😗👌

**Comment by varlogtim** [comment]
@vishnu2kmohan I think some how, through force pushing changes, your last suggestion was lost. Could you re-review? Also, I would like to get your thoughts on Eric's comment.

File: docs/how-to/installation/setup-eks-cluster.txt (hunk 3 above)
Comment (varlogtim) [reply docs/how-to/installation/setup-eks-cluster.txt]: Done!

**Comment by vishnu2kmohan** [comment]
FYI - I chatted to @eecsliu about improving our release process to add/publish a `latest` (via a symlink to the) helm chart so we don't have to constantly bump the version in the docs when there's an update (similar to [this](https://github.com/determined-ai/determined/issues/2630) - cc @dzhu).

**Comment by varlogtim** [comment]
> FYI - I chatted to @eecsliu about improving our release process to add/publish a latest (via a symlink to the) helm chart so we don't have to constantly bump the version in the docs when there's an update (similar to this - cc @dzhu).
Got it. I think I should be able to add this into the review today and remove the other code. This is a much better solution, in my opinion.

**Review by vishnu2kmohan — APPROVED** [review]

**Pull request merged**